lib/puppet/pops/issues.rb

```diff
@@ -153,28 +153,28 @@ def self.hard_issue(issue_code, *args, &block)
   # @todo configuration if this is error or warning
   #
   VAR_WITH_HYPHEN = issue :VAR_WITH_HYPHEN, :name do
-    _("A variable name may not contain a hyphen. The name '%{name}' is not legal") % { name: name }
+    _("A variable name may not contain a hyphen. The name '%{name}' is not legal.") % { name: name }
   end
 
   # A class, definition, or node may only appear at top level or inside other classes
   # @todo Is this really true for nodes? Can they be inside classes? Isn't that too late?
   # @api public
   #
   NOT_TOP_LEVEL = hard_issue :NOT_TOP_LEVEL do
-    _("Classes, definitions, and nodes may only appear at toplevel or inside other classes")
+    _("Classes, definitions, and nodes may only appear at the top level or inside other classes.")
```

Comment:
move "only" after "appear" (...nodes may appear only at the top level or...)

Reply:
updated in latest commit.